meshnesh/react-consume-api
# Ft-Auth_edit-Event

# What does this PR do?

This PR allows the owner of an event to be able to edit his events

## How should this be manually tested?

Clone the `repo`.
Install the packages `npm install` then run `npm run dev` to run the app.
Create an account on the app by `registering`, confirm your email address.
Add an event, navigate to manage events.
Click edit on the current event to edit it.

## Screenshots

<img width="1280" alt="screen shot 2018-06-04 at 12 08 30" src="https://user-images.githubusercontent.com/15923109/40908740-4934c4ee-67f0-11e8-8b2a-9536ba4aa5a3.png">
<img width="1280" alt="screen shot 2018-06-04 at 12 09 00" src="https://user-images.githubusercontent.com/15923109/40908742-495f35b2-67f0-11e8-9992-f8d8370a1f87.png">
